src/overlays/actors/ovl_En_Elf/z_en_elf.c

```diff
@@ -1083,8 +1083,10 @@ void func_80A0461C(EnElf* this, PlayState* play) {
     } else {
         arrowPointedActor = play->actorCtx.targetCtx.arrowPointedActor;
 
+        // `R_SCENE_CAM_TYPE` is checking for either `SCENE_CAM_TYPE_SHOP`, `SCENE_CAM_TYPE_FIXED`, or
+        // `SCEME_CAM_TYPE_SHOOTING_GALLERY`
```

Comment:
may be better phrased as "`R_SCENE_CAM_TYPE` is being checked for..."
in general may be worth expanding a bit on this weird usage, cause if for example anyone wants to add more types here, they will run into the issue of this bit passing the check for certain values 

something along the lines of "`R_SCENE_CAM_TYPE` is not composed of bit flags, but is being checked as one here. That makes this condition pass for a range of type values that is not explicitly clear. For example, this condition will pass for `SCENE_CAM_TYPE_SHOP`, `SCENE_CAM_TYPE_FIXED`, and `SCENE_CAM_TYPE_SHOOTING_GALLERY`

wording can be changed as seen fit, this is just a quick draft

(also `SCEME_CAM_TYPE_SHOOTING_GALLERY` -> SCENE)